programs/pluto/state.h

```diff
@@ -571,8 +571,12 @@ struct state {
 	/*
 	 * Post-quantum Preshared Key variables (v2)
 	 */
-	bool st_ppk_used;			/* both ends agreed on PPK ID and PPK */
-	bool st_v2_ike_ppk_enabled;		/* does remote peer support PPK? */
+	bool st_ppk_ike_auth_used;			/* both ends agreed on PPK ID and PPK (RFC 8784) */
+	enum {
+		PPK_DISABLED = 0,
+		PPK_IKE_AUTH = 1,		/* RFC 8784 */
+		PPK_IKE_INTERMEDIATE = 2,	/* draft-ietf-ipsecme-ikev2-qr-alt-00 */
+	} st_v2_ike_ppk;
```

Comment:
If draft 3 contains substantive changes an option is to just factor out this change so it can be merged.
Up to you.

Reply:
I've updated the PR to qr-alt-04 version in 67796a7; no structural changes, just Notify rename